Docs/reference/content/what_is_new.md

```diff
@@ -8,6 +8,12 @@ title = "What's New"
   pre = "<i class='fa fa-star'></i>"
 +++
 
+## What's New in 2.10.0-beta1
```

Comment:
It probably doesn't matter since the page gets rendered but do we want a blank line after the header?

Reply:
Added!